test/extensions/filters/http/transformation/body_header_transformer_test.cc

```diff
@@ -36,7 +36,7 @@ TEST(BodyHeaderTransformer, transform) {
 
   BodyHeaderTransformer transformer;
   NiceMock<Http::MockStreamDecoderFilterCallbacks> filter_callbacks_{};
-  transformer.transform(headers, body, filter_callbacks_);
+  transformer.transform(headers, &headers, body, filter_callbacks_);
```

Comment:
can we rename the `headers` variables here to request and response headers, so it's easier to read?

Reply:
in here they are both request headers